lib/cext/ruby/ruby.h

```diff
@@ -1519,7 +1519,7 @@ char RB_NUM2CHR(VALUE x);
 #define RB_CHR2FIX(x) INT2FIX((long)((x)&0xff))
 
 VALUE LONG2NUM(long value);
-VALUE ULONG2NUM(unsigned long value);
+#define ULONG2NUM(x) RB_ULONG2NUM(x)
```

Comment:
Note that RUBY_FIXNUM_MAX should be LONG_MAX for us, although it works anyway for this case.